pkgs/top-level/qscintilla-pyqt5-widgets.patch

```diff
@@ -0,0 +1,12 @@
+diff -u -r orig/Python/configure.py QScintilla_gpl-2.10.3/Python/configure.py
```

Comment:
This definitely cannot be in top-level. Should go to `pkgs/development/python-modules/qscintilla/`.

Reply:
whoops! editor goof